src/services/item/plugins/app/index.ts

```diff
@@ -117,6 +117,15 @@ const plugin: FastifyPluginAsync<AppsPluginOptions> = async (fastify, options) =
         return aS.getAllApps(member, buildRepositories(), publisherId);
       });
 
+      fastify.get(
+        '/most-used',
+        { schema: getMany },
+        async ({ member, authTokenSubject: requestDetails }) => {
+          const memberId = member ? member.id : requestDetails?.memberId;
+          return aS.getMostUsedApps(member, buildRepositories(), memberId as string);
```

Comment:
I don't think the casting should be done here, instead check for `memberId` and if it is not available throw an error as this endpoint does not work if users are not authenticated.